slides/developer-guide/working-with-masterslides/read-information-of-master-slides-from-a-powerpoint-presentation/_index.md

```diff
@@ -0,0 +1,296 @@
+---
+title: "Read Information of Master Slides from a PowerPoint Presentation"
+type: docs
+url: /read-information-of-master-slides-from-a-powerpoint-presentation/
+weight: 10
+---
+
+## **Introduction**
+
+Master slides contain the same fonts and images in one place for all slides in your presentation. When you edit a master slide, all slides that are based on that master slide will contain those changes. The following method allows you to read information of all master slides from a PowerPoint presentation.
```

Comment:
Cannot figure out the meaning of this sentence:
"Master slides contain the same fonts and images in one place for all slides in your presentation."
I think it should be rephrased, perhaps just exclude the words "the same"

Reply:
This phrase was borrowed from Microsoft documentation. It is correct.